README.md

```diff
@@ -0,0 +1,38 @@
+# C2 Malware Traffic Response
+
+## Overview
+
+### Introduction
+
+This solution pack demonstrates the C2 Attacks, in such attacks, the malware-infected system establishes a connection with the attacker's server (C2 server) to maintain communication with the infected system. The attacker uses such attacks to gain control over the system, data theft or download other malicious software.
```

Comment:
Should we add meaning of C2?

Reply:
Where is details of scenario? Please check phishing mail scenario doc as a reference.

Reply:
Updated the introduction. Please check @cs-suyog-jinturkar